src/makeRobot.js

```diff
@@ -38,7 +38,72 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    version,
+    wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+      + `chip version: ${this.version}, `
+      + `wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    evacuate() {
+      this.coords.x = 1400;
+      this.coords.y = 500;
+    },
+
+    goForward(y = 1) {
+      if (y < 0) {
+        return this;
+      }
+
+      this.coords.y += y;
```

Comment:
The second return is redundant here, you can do some logic if y > 0, and at the end of the method return this
```suggestion
      if (y > 0) {
        this.coords.y += y;
      }
```